Bottom: 97e8b521eeff84d3264fb4dcbfd9ef38e02d0a7b
Top:    e2890710c970dd207d6637b2e82f3ca0c5c38f25
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-05-11 15:35:08 +0800

Refresh of vhost_net-add-interrupt

---

diff --git a/drivers/vhost/net.c b/drivers/vhost/net.c
index 80d9f6c..ac24979 100644
--- a/drivers/vhost/net.c
+++ b/drivers/vhost/net.c
@@ -639,6 +639,7 @@ static void handle_rx(struct vhost_net *net)
 		}
 	}
 out:
+	vhost_check_coalesce_and_signal(dev, vq);
 	mutex_unlock(&vq->mutex);
 }
